test/blackbox-tests/test-cases/pkg/conflicts.t

```diff
@@ -4,7 +4,7 @@ The solver should repsect the (conflicts) field of the (package) stanza.
   $ mkrepo
   $ mkpkg foo
   $ mkpkg bar << EOF
-  > depends: [ foo ]
+  > depends: [ "foo" ]
   > EOF
 
 The solver should say no solution rather than just ignoring the conflict.
```

Comment:
This comment needs to be updated also then.

Reply:
Why, the comment is still correct. The conflict is ignored.